examples/config-policy.json

```diff
@@ -0,0 +1,76 @@
+{
+  "version":"0.1.0-dev",
+  "storage":{
+    "rootDirectory":"/tmp/zot"
+  },
+  "http": {
+    "address":"127.0.0.1",
+    "port":"8080",
+    "realm":"zot",
+    "tls": {
+      "cert":"test/data/server.cert",
+      "key":"test/data/server.key"
+    },
+    "auth": {
+      "ldap": {
+        "address":"ldap.example.org",
+        "port":389,
+        "startTLS":false,
+        "baseDN":"ou=Users,dc=example,dc=org",
+        "userAttribute":"uid",
+        "bindDN":"cn=ldap-searcher,ou=Users,dc=example,dc=org",
+        "bindPassword":"ldap-searcher-password",
+        "skipVerify":false,
+        "subtreeSearch":true
+      },
+      "htpasswd": {
+        "path": "test/data/htpasswd"
+      },
+      "failDelay": 5
+    },
+    "accessControl": {
+      "defaults": [
+        {
+          "users": ["alice", "bob"],
+          "actions": ["read", "write"]
+        },
+        {
+          "users": ["*"],
+          "actions": ["read"]
+        }
+      ],
+      "repositories": [

```

Comment:
can we add everything on repositories level, if repo is present in list then it will follow those rules and if repo not present in list it will follow whatever default we decides. 

``` json
 "repositories":
  [
       {
 		"name": "repos/repo1",
 		"actions": {
 			"read": ["alice", "bob"],
 			"write": ["alice", "bob"]
 		}
 	},
 	{
 		"name": "repos/repo2",
 		"actions": {
 			"read": ["*"],
 			"write": ["carlos"]
 		}
 	},
 	 {
 		"name": "repos/repo3",
 		"actions": {
 			"read": ["alice", "bob"],
 		}
 	 }
 ]
  ```